subsys/net/ip/ipv6_mld.c

```diff
@@ -124,16 +124,19 @@ static int mld_create_packet(struct net_pkt *pkt, uint16_t count)
 
 static int mld_send(struct net_pkt *pkt)
 {
+	int res;
```

Comment:
Please use `ret` as a variable name (just to be consistent with other functions in this file)